src/views/design/ComponentDetail/index.tsx

```diff
@@ -800,6 +806,33 @@ function mapComponentTypeToListItems(items: IComponentType[]) {
     return listItems;
 }
 
+function orderParameter(items: IComponentParameter[], componentType: IComponentType) {
```

Comment:
rename to orderComponentParameters.
Also, define the return type of the function